arsceneview/src/main/java/io/github/sceneview/ar/interaction/ArNodeManipulator.kt

```diff
@@ -69,7 +65,7 @@ open class ArNodeManipulator(
             currentGestureTransform == EditableTransform.POSITION && it.positionEditable
         }?.let { node ->
             currentGestureTransform = null
-            (node as? ArModelNode)?.updateUnanchoredPose = savedUnanchoredUpdateState
+            (node as? ArModelNode)?.followHitPosition = savedFollowHitPositionState
```

Comment:
Should be called after the `anchor()`

Reply:
I've moved the statement, however, since everything runs on the main thread the order doesn't really matter now.

Reply:
Sure, but it's better for logical reasons.